recipes/libplist/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibplistConan(ConanFile):
+    name = "libplist"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libimobiledevice-win32/libplist"
+    description = "A library to handle Apple Property List format in binary or XML"
+    topics = ["conan", "plist", "apple"]
+    license = "LGPL-2.1"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "libplist"
```

Comment:
```suggestion
        return "source_subfolder"
```

For consistency with others CCI recipes.

Reply:
fixed